src/config/env/dev.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "VITE_REACT_APP_DCL_SQUID_MANAGEMENT_SERVER": "https://squid-management-server.decentraland.org"
```

Comment:
I think it's not required to use the prefix `VITE_REACT_APP_DCL` for the config var names